### PR TITLE
#1366 Increased comment length

### PR DESCRIPTION
**Pull request by arvind-1**
https://github.com/elimu-ai/webapp/issues/1366

### CODE REVIEW

**Review by nya-elimuai — CHANGES_REQUESTED** [review]
@arvind-1 Thank you, these changes look good. Also, you will have to add a database migration script that will update the current database structure to the new column type. You can see a similar example here: https://github.com/elimu-ai/webapp/commit/b63a73a3234c657a765e04df1c8c368f72f7015e#diff-ef68d66d4f2c00c95c2b813cbd433318759ff62453681f95dfacb0c61fa15f26

Also, please add a `maxlength="1000"` attribute to the `edit.jsp`, as well as a maxlength check in the corresponding back-end controller (`StringUtils.abbreviate(comment, 1000)`).

CC @Shubham-Vishwakarma

File: src/main/java/ai/elimu/rest/v2/crowdsource/WordContributionRestController.java [hunk 1]
```diff
@@ -156,7 +156,7 @@ public String postWordContribution(
             wordContributionEvent.setTime(wordContributionEventGson.getTime());
             wordContributionEvent.setWord(word);
             wordContributionEvent.setRevisionNumber(word.getRevisionNumber());
-            wordContributionEvent.setComment(wordContributionEventGson.getComment());
+            wordContributionEvent.setComment(StringUtils.abbreviate(wordContributionEventGson.getComment(), 1000));
```
Comment (nya-elimuai) [inline src/main/java/ai/elimu/rest/v2/crowdsource/WordContributionRestController.java:159]: @arvind-1 Thanks for remembering to include the change for the REST controller, I forgot about that. 🙂

Can you also implement the same change in the `WordCreateController` and `WordEditController`?

**Review by nya-elimuai — CHANGES_REQUESTED** [review]

**Comment by arvind-1** [comment]
@nya-elimuai Added all the required file now. Thanks for guiding.

File: src/main/resources/db/migration/2002004.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# 2.2.4
+
+# Increase comment length from 255 to 1000
+ALTER TABLE WordContributionEvent MODIFY `comment` varchar(1000);
```
Comment (nya-elimuai) [inline src/main/resources/db/migration/2002004.sql:4]: @arvind-1 Please add a new line at the end of this file.

**Review by nya-elimuai — CHANGES_REQUESTED** [review]
Migration script is missing a newline character.

**Review by nya-elimuai — APPROVED** [review]
Looks good. Ready to be merged 👍

**Pull request merged**